### PR TITLE
chore(iframe): try/catch access to the top frame (2.x)

### PR DESCRIPTION
**Pull request by halkon**
JIRA:

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] QE LGTM

### CODE REVIEW

**Comment by lentusignavus** [comment]
LGTM

File: src/components/rxApp/scripts/rxApp.js [hunk 1]
```diff
@@ -44,7 +44,17 @@ angular.module('encore.ui.rxApp')
...
             scope.isWarning = scope.isPreProd || scope.isLocalNav;
 
-            scope.isEmbedded = $window.self !== $window.top;
+            scope.isEmbedded = false;
+            try {
+                // Checks to see if we have access to the global scope of a DOM Window
+                // Element by attempting to set a property on it.  If we have no errors
+                // then this means that `document.domain` matches and we have no Cross
+                // Origin security constraints
+                $window.top['encore'] = true;
```
Comment (Droogans) [inline src/components/rxApp/scripts/rxApp.js]: https://github.com/rackerlabs/encore-ui/pull/1822#discussion_r80506093 applies here as well.

Comment (halkon) [inline src/components/rxApp/scripts/rxApp.js]: I'll update these two with all the final changes of #1822

Comment (halkon) [inline src/components/rxApp/scripts/rxApp.js]: Done

**Comment by CITguy** [comment]
@Droogans: you good with this PR?

**Pull request merged**